src/main/java/net/minecraftforge/common/lighting/LightingHooks.java

```diff
@@ -38,9 +38,12 @@ public static void relightSkylightColumn(final World world, final Chunk chunk, f
         final int xBase = (chunk.xPosition << 4) + x;
         final int zBase = (chunk.zPosition << 4) + z;
 
-        for (int y = yMax; y >= yMin; --y)
+        int y;
+        ExtendedBlockStorage section = null;
```

Comment:
why use `null `here and `Chunk.NULL_BLOCK_STORAGE` later when checking for null?